src/media/css/feed/feed.styl

```diff
@@ -222,35 +174,79 @@
                 margin-right: 0;
             }
         }
-        + .curve {
+        + .feed-item-curve {
+            height: 60px;
             top: 30px;
         }
     }
-    &.full.has-background .info {
-        margin-top: 83px;
-
-        .desc {
-            padding: 10px;
-        }
+    &.has-desc .feed-item-curve {
+        height: 60px;
+        top: 75px;
     }
-    &.full.has-background {
-        .info {
-            position: relative;
-            top: 10px;
-            z-index: 15;
-        }
-        .curve {
-            top: 75px;
-        }
-        .curve-bottom {
-            height: 200px;
-            position: absolute;
-            top: 60px;
-            width: 400px;
-        }
+    .app-icons + .feed-item-curve {
+        top: 30px;
     }
 }
 
+// Promo tile classes.
+.feed-item-info {
```

Comment:
should we call these feed-promo-tile and move them into maybe a ```promo-tile.styl``` file? These don't apply to all feed items right?